src/reanimated2/animation/repeat.ts

```diff
@@ -39,6 +39,11 @@ export const withRepeat = function <T extends AnimationObject>(
 ): Animation<RepeatAnimation> {
   'worklet';
 
+  const initialAnimationStart =
+    typeof _nextAnimation === 'function'
+      ? _nextAnimation().startValue
+      : _nextAnimation.startValue;
+
```

Comment:
// This is code from lines 52-55, it can be removed from lines 52-55 and placed here
  const nextAnimation =
    typeof _nextAnimation === 'function'
      ? _nextAnimation()
      : _nextAnimation;
          
  const initialAnimationStart = nextAnimation.startValue;